tools/bridge/bridge/event_fetcher.py

```diff
@@ -1,12 +1,17 @@
 import logging
-from time import sleep
+from time import sleep, time
 from typing import Any, Dict, List
 
 from eth_utils import to_checksum_address
 from web3 import Web3
 from web3.contract import Contract
 
 
+class FetcherReachedHeadEvent:
+    def __init__(self):
+        self.timestamp = int(time())
```

Comment:
what's the reason to not use `float` here?

Reply:
I was under the false impression we need the full seconds here. I just remove that. :laughing:

Reply:
:heavy_check_mark: